### DISCUSSION TITLE
New pre-release of mu-argus for testing (version 5.1.7)

### DISCUSSION THREAD

**Discussion by ppdewolf** [sdcTools/UserSupport#226] [User Group]
Dear all,

_Even though we do have a testers team, I include this information for other Users as well._

I have just uploaded a new version of mu-argus ([version 5.1.7](https://github.com/sdcTools/muargus/releases/tag/v5.1.7)) to be tested. The main addition is that now multiple similarity profiles are possible for Targeted Record Swapping.

So I would appreciate your feedback on this new feature. And your check on whether the "old" features are still functioning as they should ;-)

Even though the summer holidays are near, I would appreciate your feedback before 1 August 2022. After that I will make the release official.

Thanks!
Peter-Paul

**Comment by ppdewolf** [comment]
For those of you who would like to test mu-argus 5.1.7:
I've just uploaded a [new build](https://github.com/sdcTools/muargus/releases/tag/v5.1.7b3) with some bug fixes, found by some testers (in the "MakeProtectedFile" window you could not select all suppression methods).

[Version 5.1.7build3](https://github.com/sdcTools/muargus/releases/tag/v5.1.7b3)

**Comment by tdlinehancso** [comment]
Peter,

Thanks,

Tim

"Peter-Paul de Wolf" ---26/07/2022 13:38:44---CAUTION: This email originated from outside of the CSO. Do not click links or open   attachments unl

From:	"Peter-Paul de Wolf" ***@***.***>
To:	
Cc:	"Subscribed" ***@***.***>
Date:	26/07/2022 13:38
Subject:	Re: [sdcTools/user-group] New pre-release of mu-argus for testing (version 5.1.7) (#21)






CAUTION: This email originated from outside of the CSO. Do not click links or open attachments unless you recognise the sender and know the content is safe. If in doubt contact the Service Desk at Extension 5666.





For those of you who will would like to test mu-argus 5.1.7:
I've just uploaded a new build with some bug fixes, found by some testers (in the "MakeProtectedFile" window you could not select all suppression methods).
Version 5.1.7build3
